templates/sections/events-editorial-content.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
```

Comment:
@jenbreese is this template needed? It looks like you could do most of this with the jumpstart-ui one column layout instead of using the two column layout and removing a column.

Reply:
@sherakama, I was able to do it with the layout builder. I will delete the template?

Reply:
If you don't need the template, yes please, remove it.

Reply:
I deleted the extra template.